libeos-payg/manager-service.c

```diff
@@ -835,16 +835,17 @@ epg_manager_service_manager_add_code (EpgManagerService     *self,
                                       GDBusMethodInvocation *invocation)
 {
   g_autoptr(GError) local_error = NULL;
+  gint64 time_added = NULL;
```

Comment:
`NULL` is not a `gint64`. You want `0`.

Reply:
Is `-Wint-conversion` not enabled, or is someone ignoring warnings?